assets/js/custom.js

```diff
@@ -412,5 +412,18 @@ $(function() {
 	});
 });
 
+$(function() {
+  /* Carousel header swipe on mobile */
+  var carousel_element = document.getElementsByClassName('carousel-header')[0];
```

Comment:
@ashish173 @comzeradd This line causes js error
`Uncaught TypeError: Cannot read property 'addEventListener' of undefined`
in search page breaking the rest of the js functionality in the page
https://dev.p4.greenpeace.org/international/?s=&orderby=relevant

Reply:
@ashish173 you can probably use jquery to do the selection here and then check `carousel_header.length` to run the rest of the code.

Reply:
Thanks @angtheod you are the real bug catcher; @comzeradd thanks for the suggestion #262 is the fix for it.